configure.ac

```diff
@@ -1923,6 +1965,7 @@ pcap_sendpacket:            ${have_pcap_sendpacket} **
 pcap_netmap                 ${have_pcap_netmap}
 Linux/BSD netmap:           ${have_netmap}
 Tuntap device support:      ${have_tuntap}
+LIBXDP for AF_XDP socket:   ${have_libxdp}
```

Comment:
At the end of ./configure there is a summary table, I appended it as well with the result of checking  libxdp library